src/main/java/wolox/training/models/Book.java

```diff
@@ -51,6 +53,10 @@
     @Column(nullable = false, unique =true)
     private String isbn;
 
+    @ManyToOne
+    @JoinColumn(name = "user_id")
```

Comment:
This relation should be @ManyToMany(mappedBy = books), and you can delete @JoinColumn(..)

Reply:
I made this change